JsonLd/ResourceInterface.php

```diff
@@ -64,6 +64,20 @@ public function initFilters(array $filters);
     public function getFilters();
 
     /**
+     * Initializes order filters.
+     *
+     * @param array $orders
+     */
+    public function initOrder(array $orders);
```

Comment:
Same here `$order` or `initOrders` (coherence). I've a preference for `initOrders` and `getOrders`.